crates/uv/src/commands/venv.rs

```diff
@@ -111,6 +114,16 @@ async fn venv_impl(
     )
     .into_diagnostic()?;
 
+    if path.exists() && !clear {
+        writeln!(
+            printer,
+            "Virtualenv already exists at {}. Use --clear option to recreate",
+            path.normalized_display().cyan()
+        )
+        .into_diagnostic()?;
+        return Ok(ExitStatus::Success);
+    }
+
```

Comment:
I'd rather to fail with non-zero exit code.
Also, message could be more strict since the command failing:
```
Failed to create virtualenv at '{}': Already exists.
hint: Use --clear option to force recreate new virtualenv at {}
```